certify-service/src/main/java/io/mosip/certify/entity/attributes/CredentialStatusDetail.java

```diff
@@ -0,0 +1,24 @@
+package io.mosip.certify.entity.attributes;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CredentialStatusDetail implements Serializable {
+    @JsonProperty("status_purpose")
+    private String statusPurpose;
+    @JsonProperty("status_value")
+    private Boolean statusValue;
```

Comment:
Critical: What is the significace of this field, since it will not have the updated status anyways ?